src/index.html

```diff
@@ -1,13 +1,410 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Miami Condo Kings</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/favicon.png"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <!--  HEADER -->
+    <header class="header page__section">
+      <div class="header__content">
+        <div class="header__top-nav">
+          <label
+            class="icon icon--menu"
+            for="menu-checkbox"
+          >
+          </label>
+
+          <!--   LOGO   -->
+          <a class="logo" href="/">
+            <img
+              class="logo__img"
+              src="./images/logo.png"
+              alt="Miami Condo Kings"
+            >
+          </a>
+
+          <a class="icon icon--tel" href="tel: +1 234 555-55-55">
+            <div class="icon__tooltip">+1 234 555-55-55</div>
+          </a>
```

Comment:
It was possible to create a tooltip using pseudo-elements, without additional tags in html.